src/main/scala/cognite/spark/v1/NumericDataPointsRdd.scala

```diff
@@ -337,7 +337,9 @@ case class NumericDataPointsRdd(
       .parEvalMapUnordered(50) {
         case (id, Some(first), Some(latest)) =>
           val aggStart = Instant.ofEpochMilli(floorToNearest(first.toEpochMilli, granularityMillis))
-          val aggEnd = Instant.ofEpochMilli(ceilToNearest(latest.toEpochMilli, granularityMillis))
+          val aggEnd = Instant
+            .ofEpochMilli(ceilToNearest(latest.toEpochMilli, granularityMillis))
+            .plusMillis(1) //endpoint is exclusive
```

Comment:
`// end is exclusive` (to match the name `end`) in https://docs.cognite.com/api/v1/#operation/getMultiTimeSeriesDatapoints